onnxruntime/core/graph/graph.cc

```diff
@@ -651,7 +653,13 @@ Graph::Graph(GraphProto* graph_proto, const std::unordered_map<std::string, int>
     // Copy constant nodes _value to name_to_initial_tensor_
     const gsl::not_null<TensorProto*>
         tensor{graph_proto_->add_initializer()};
-    *tensor = node.attribute(0).t();
+    const AttributeProto& constant_attribute = node.attribute(0);
+    // TODO: Add support for parsing 'sparse_value' attribute from a 'Constant' node
+    // Discussion surrounding handling the SparseTensorProto must be had. 
+    // An easy way is to implement a method that converts a SparseTensorproto into a TensorProto 
+    // to use the same downstream flow, but that is going to impact peak memory usage and probably a smarter way is required.
+    ORT_ENFORCE(constant_attribute.has_t(), "Only 'value' attribute is supported within a 'Constant' node in ORT");
```

Comment:
@skottmckay - persuant to our conversation, removed the attribute size check that basically prevents an invalid model (with an invalid Constant node) sneaking in, still kept the enforcement that the attribute that we extract from the 'Constant' node has a TensorProto.